duckduckgo/iOS
# Change save password Never for Site button to Not Now

Task/Issue URL: https://app.asana.com/0/1202926619870900/1208592991532541/f

**Description**:

Change the Never Save button with Not Now on mobile platforms to avoid users accidentally disabling for their top sites on first usage.
Update the logic accordingly.
Send dismiss pixel on press of the Never Save button.

**Steps to test this PR**:
1. Clear your autofill data from the debug menu
2. Go to https://fill.dev/form/login-simple and submit some details.
3. **Make sure there is no "Never Ask for This Site" button and there is a "Not Now" button**
4. Tap the "Not Now" button.
5. Repeat step 2.
6. **Make sure the Save prompt shows again**

**Definition of Done (Internal Only)**:


**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
